README.md

```diff
@@ -1,68 +1,32 @@
-# MinIO Operator Guide [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/k8s-operator.svg?maxAge=604800)](https://hub.docker.com/r/minio/k8s-operator)
+# MinIO Operator [![Docker Pulls](https://img.shields.io/docker/pulls/minio/k8s-operator.svg?maxAge=604800)](https://hub.docker.com/r/minio/k8s-operator)
 
-MinIO is a high performance distributed object storage server, designed for large-scale private cloud infrastructure. MinIO is designed in a cloud-native manner to scale sustainably in multi-tenant environments. Orchestration platforms like Kubernetes provide perfect launchpad for MinIO to scale.
+MinIO Operator brings native support for [MinIO](https://github.com/minio/minio), [Console](https://github.com/minio/console), and [KES](https://github.com/minio/kes) to Kubernetes.
 
-MinIO-Operator brings native MinIO, [Console](https://github.com/minio/console), and [KES](https://github.com/minio/kes) support to Kubernetes. MinIO-Operator currently supports following features:
+## Prerequisites
 
-| Feature                 | Reference Document |
-|-------------------------|--------------------|
-| Create and delete highly available distributed MinIO clusters  | [Create a MinIO Instance](https://github.com/minio/operator#create-a-minio-instance). |
-| TLS Configuration  | [TLS for MinIO Instance](https://github.com/minio/operator/blob/master/docs/tls.md). |
-| Expand an existing MinIO cluster | [Expand a MinIO Cluster](https://github.com/minio/operator/blob/master/docs/adding-zones.md). |
-| Use a custom template for hostname discovery | [Custom Hostname Discovery](https://github.com/minio/operator/blob/master/docs/custom-name-templates.md). |
-| Use PodSecurityPolicy for MinIO Pods | [Apply PodSecurityPolicy](https://github.com/minio/operator/blob/master/docs/pod-security-policy.md). |
-| Deploy Console with MinIO cluster  | [Deploy MinIO Instance with Console](https://github.com/minio/operator/blob/master/docs/console.md). |
-| Deploy KES with MinIO cluster  | [Deploy MinIO Instance with KES](https://github.com/minio/operator/blob/master/docs/kes.md). |
+- Kubernetes >= v1.17.0.
+- Create PVs using [direct CSI driver](https://github.com/minio/operator/blob/master/docs/using-direct-csi.md).
+- Install `kubectl minio` plugin.
 
-## Getting Started
+## Steps to Run
 
```

Comment:
Should state explicitly up here what this command does:

```
The following command creates a distributed MinIO deployment with 4 MinIO ``servers``. Each MinIO server consists of four ``volumes`` with a ``capacity`` of 1Ti each. 
```

You can follow up noting something like

```
To modify the number of servers, volumes, or capacity per volume, change the values for ``servers``, ``volumes``, and
``capacity`` respectively. 

- MinIO Erasure Coding relies on at least 4 drives across all MinIO instances in the distributed system. If the value of 
  ``servers X volumes`` is less than ``4``, MinIO cannot enable erasure coding on the deployment.

- The MinIO operator cannot expand standalone deployments (``servers 1``) to distributed deployments.

Reply:
I forgot to ask this during the last engineering meeting - I remember vaguely we had some docs noting that we cannot expand from standalone to distributed. Maybe that's in my imagination though - I don't see it anymore (perhaps its no longer a concern). If so can scratch the last bullet.